src/httpserver.cpp

```diff
@@ -19,6 +19,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <signal.h>
+#include <future>
```

Comment:
I guess I should research <functional> and <future> before claiming trivial ACK....